docs/get_started/quickstart.rst

```diff
@@ -8,32 +8,359 @@ Quick Start
 =====================
 
 |productName| has a variety of APIs to choose from when setting up and running a federation. 
-In this quick start guide, we will demonstrate how to run a simple federated learning example using the Task Runner API and Hello Federation script
+In this quick start guide, we will demonstrate how to run a simple federated learning example using the Task Runner API.
 
-.. note::
 
-    The example used in this section is designed primarily to demonstrate functionality of the package and its components. It is not the recommended method for running a real world federation.
 
-    See :ref:`openfl_examples` for details.
+.. _creating_a_federation:
 
-.. _hello_federation:
+********************************
+Creating a federation in 5 steps
+********************************
 
-*********************************
-Hello Federation
-*********************************
-.. note::
+To begin we recommend installing OpenFL inside a python virtual environment. This can be done with the following:
 
-    Ensure you have installed the |productName| package.
+.. code-block:: console:
+    
+    pip install virtualenv
+    virtualenv ~/openfl-quickstart

```

Comment:
I almost always suggest specific python versions that we are sure/tested against.
```virtualenv -p /usr/bin/python3.10 ~/openfl-quickstart/bin/activate```